timetomodel/speccing.py

```diff
@@ -569,3 +654,21 @@ def parse_series_specs(
         return ObjectSeriesSpecs(specs, name)
     else:
         return specs
+
+
+def pass_or_set_default_resampling_config(
```

Comment:
Difficult to find a good name here. I don't know what you meant with "pass"...
I believe a better name is `ensure_resampling_config_is_set`.